app/views/kaminari/_paginator.html.haml

```diff
@@ -1,5 +1,5 @@
 = paginator.render do
-  %ul.pager-items
+  %ul.pager-items{"data-click-events" => "", "data-click-category" => "Register Table", "data-click-action" => "Pagination"}
```

Comment:
does this empty string `"data-click-events" => ""` do anything, can we just remove it?

Reply:
Initializes the JS 👍